### PR TITLE
[el_vietnamese_viqr] Vietnamese-VIQR Keyboard

### PR DESCRIPTION
**Pull request by MakaraSok**

### CODE REVIEW

**Comment by MakaraSok** [comment]
The contents of the welcome.htm file from the .kmp of the previous version have been included in the now welcome.htm file.

**Comment by MakaraSok** [comment]
@DavidLRowe Could you please review this keyboard and label it as 'ready to merge' if everything is up to par?

**Comment by mcdurdin** [comment]
Versioning questions need to be resolved before we merge this keyboard. There may be an existing newer binary version.

File: release/el/el_vietnamese_viqr/source/el_vietnamese_viqr.kps [hunk 1]
```diff
@@ -69,5 +45,15 @@
       <FileType>.kmx</FileType>
     </File>
   </Files>
+  <Keyboards>
+    <Keyboard>
+      <Name>OR Vietnamese (VIQR)</Name>
+      <ID>el_vietnamese_viqr</ID>
+      <Version>1.1</Version>
+      <Languages>
+        <Language ID="vi">vi</Language>
```
Comment (mcdurdin) [inline release/el/el_vietnamese_viqr/source/el_vietnamese_viqr.kps]: The language name should be "Vietnamese", not "vi" (ID is correct).

**Pull request closed without merging**